discovery/pkg/openstack/translate.go

```diff
@@ -114,6 +114,15 @@ func serviceName(lb loadbalancers.LoadBalancer) string {
 	return strings.ToLower(lbName)
 }
 
+// get the lb Name or ID if name is empty
+func serviceNameOriginal(lb loadbalancers.LoadBalancer) string {
```

Comment:
Will this be a problem if there are two load balancers with the same name? Seems like it'll affect the metrics, but not 100% sure.

Reply:
Yes, it will. @rosskukulinski do you want to allow duplicates here? It only should affect openstack at the moment.

Reply:
The alternative is to use the name + ID, but not sure if that is what we want.

Reply:
TBH I'm really not sure. I don't know how often folks name their load balancers the same thing in the same cluster.  Let's leave it as-is for now (at risk of duplicates).

If the LB doesn't have a name, do we put in the id for the label?

Reply:
yes name is tried first, then id is used.